paddle/fluid/extension/include/dtype.h

```diff
@@ -13,27 +13,91 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 #pragma once
+
 #include "paddle/fluid/platform/bfloat16.h"
 #include "paddle/fluid/platform/complex128.h"
 #include "paddle/fluid/platform/complex64.h"
 #include "paddle/fluid/platform/float16.h"
 
 namespace paddle {
 
+using float16 = paddle::platform::float16;
```

Comment:
remove this, i will using pre-declaration to do the same thing and hide all these headers

Reply:
only forward declare is not enough, when users access data, it will throw error